app/views/smart_proxies/plugins/_dhcp_reservations.html.erb

```diff
@@ -0,0 +1,18 @@
+<table id="<%= subnet.network.parameterize %>_reservations" class='<%= table_css_classes %> table-fixed' data-table="inline">
+  <thead>
+    <tr>
+      <th class="col-md-4"><%= _("Hostname") %></th>
+      <th class="col-md-2"><%= _("IP Address") %></th>
+      <th class="col-md-2"><%= _("MAC") %></th>
```

Comment:
It displays ok for me: 
![](http://i.imgur.com/fwNs6P4.png)
but you are probably right, I will make it add up to 12 just to be on a safe side.